smart_ats/notifications/core.py

```diff
@@ -8,7 +11,12 @@ def send(data: str):
 
 class EmailNotification:
     @staticmethod
-    def send(data: str):
+    def send(data: str, email: str) -> None:
         # call sms service call sendgrid
-        print("=======EMAIL============")
-        print(data)
+        send_mail(
```

Comment:
move to dataclass and avoid hard coding anything here.
act as library, make dynamic as possible.